concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -168,3 +198,26 @@ def test_download_should_return_401_if_wrong_token_cluster_address(self):
         self.assertIsInstance(response, JsonResponse)
         self.assertEqual(response.status_code, 401)
         self.assertIn('message', response.json().keys())
+
+    @freeze_time("2018-12-30 11:00:00")
+    def test_download_should_return_401_if_file_paths_are_not_unique(self):
+        self.upload_token.files.append(FileTransferToken.FileInfo())
+        self.upload_token.files[1]['path']      = 'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
+        self.upload_token.files[1]['checksum']  = 'sha1:098f6bcd4621d373cade4e832627b4f6'
+        self.upload_token.files[1]['size']      = 1024
+
+        golem_upload_token = dump(self.upload_token, settings.CONCENT_PRIVATE_KEY, settings.CONCENT_PUBLIC_KEY)
+        encrypted_token = b64encode(golem_upload_token).decode()
+        response = self.client.get(
+            '{}{}'.format(
+                reverse('gatekeeper:download'),
+                'blender/benchmark/test_task/scene-Helicopter-27-cycles.blend'
+            ),
+            HTTP_AUTHORIZATION = 'Golem ' + encrypted_token,
+            HTTP_CONCENT_CLIENT_PUBLIC_KEY = self.public_key
+        )
+
+        self.assertIsInstance(response, JsonResponse)
+        self.assertEqual(response.status_code, 401)
+        self.assertIn('message', response.json().keys())
+        self.assertIn("application/json", response["Content-Type"])
```

Comment:
Shouldn't it be equal ?

Reply:
Well, I not sure in this case.

Reply:
I have checked and this header is string looking exactly "application/json", so I don't see any reason to not compare it using assertEqual. If there is a reason for that, please let me know, if not, please change (in other places too).

Reply:
Done.